modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/PartitionScanPublisher.java

```diff
@@ -85,6 +86,11 @@ public void subscribe(Subscriber<? super T> subscriber) {
      */
     protected abstract CompletableFuture<Void> onClose(boolean intentionallyClose, long scanId, @Nullable Throwable th);
 
+    @TestOnly
+    public long scanId(Subscription subscription) {
```

Comment:
Probably, it makes sense to mention that this new method can be removed when the colocation track is implemented.

Reply:
I'd even add todo with our colocation cleanup ticket.

Reply:
Agree, would add TODO with https://issues.apache.org/jira/browse/IGNITE-22522